src/pages/styles.module.css

```diff
@@ -32,13 +40,33 @@
 }
 
 .featureImage {
-  height: 130px;
-  width: 130px;
+  height: 90px;
+  width: 90px;
 }
 
 .featuresGrid {
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(360px, 1fr));
-  grid-auto-rows: 330px;
-  gap: 4px;
+  grid-auto-rows: 250px;
+  gap: 2.5rem;
+}
+
+/* new styles */
+
+.padmd {
+  padding: 0 2rem;
+}
+
```

Comment:
This class is used in only one place, and follows the "utility CSS" approach of styling. The pre-existing styles in this stylesheet all follow a "semantic CSS" approach of styling. I don't think we should mix these two approaches in the same stylesheet -- it introduces uncertainty for someone coming in and making new changes, about which style to follow. 

If the utility class was used many times, I might feel differently, but especially since it's only used once, I'd prefer for us to apply this specific padding in the semantic class that needs it. 

The one catch with that, in this specific example, is that the element that requires this padding applies the `.features` style......which is reused on a second different element on the page. The best way to fix this, IMO, is to create separate semantic classes for the two call sites, and duplicate the shared styles across them. Then we can apply this unique style to the one class that needs it.

This was a little hard to explain, so I'm worried it doesn't make sense. Please let me know if it doesn't!

Reply:
@pepopowitz Great review - I've changed all the CSS to use semantic references + camelCase as you suggest, and it's much better thank you!

_(I was trying to prepare for if/when we moved styles out into the main CSS and that are applied more globally if/when we start using these components in other areas, but we can tackle that if/when we actually need to and it was a bit messy 👍)_